app/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'com.google.firebase.appdistribution'
 apply plugin: 'com.github.triplet.play'
 apply plugin: 'de.mannodermaus.android-junit5'
 apply plugin: 'io.fabric'
+apply plugin: 'com.google.gms.google-services'
```

Comment:
https://developers.google.com/android/guides/google-services-plugin

>Add dependencies for basic libraries required for the services you have enabled. This step requires that the apply plugin: 'com.google.gms.google-services' line be at the bottom of your app/build.gradle file so that no dependency collisions are introduced. You can see the result of this step by running ./gradlew :app:dependencies.